helix-term/src/application.rs

```diff
@@ -845,3 +852,17 @@ impl Application {
         Ok(())
     }
 }
+
+#[cfg(not(windows))]
+fn check_enable_suspend() -> bool {
+    unsafe { libc::signal(libc::SIGTSTP, libc::SIG_IGN) == 0 }
+}
+
+#[cfg(not(windows))]
+fn get_available_signals(enable_suspend: bool) -> Result<Signals, std::io::Error> {
+    let mut app_signals = vec![libc::SIGCONT];
+    if enable_suspend {
+        app_signals.push(libc::SIGTSTP);
+    }
+    Signals::new(app_signals)
```

Comment:
just to make it a little closer to the original implementation

```suggestion
fn get_available_signals(enable_suspend: bool) -> Result<Signals, Error> {
    let mut app_signals = vec![libc::SIGCONT];
    if enable_suspend {
        app_signals.push(libc::SIGTSTP);
    }
    Signals::new(app_signals).context("build signal handler")
```